diff -Naur seed-3.8.1.orig/configure.ac seed-3.8.1/configure.ac
--- seed-3.8.1.orig/configure.ac	2015-01-08 16:54:24.000000000 +0700
+++ seed-3.8.1/configure.ac	2015-04-08 14:28:17.162944011 +0700
@@ -74,21 +74,21 @@
 AC_SUBST(FFI_CFLAGS)
 
 dnl ==============================WebKit=======================================
-m4_define([webkit_default], [3.0])
+m4_define([webkit_default], [4.0])
 
 AC_ARG_WITH([webkit],
-            [AC_HELP_STRING([--with-webkit=@<:@3.0/1.0@:>@],
-                            [Select the Webkit backend, default=3.0])],
+            [AC_HELP_STRING([--with-webkit=@<:@4.0/1.0@:>@],
+                            [Select the Webkit backend, default=4.0])],
 	    [],
             [with_webkit=webkit_default])
 
-dnl=== support both 1.0 and 3.0 (default 3.0) ==
+dnl=== support both 1.0 and 4.0 (default 4.0) ==
 AS_CASE([$with_webkit],
 	[3.0], [
-		WEBKIT_PC=webkitgtk-3.0
+		WEBKIT_PC=webkit2gtk-4.0
 		SEED_GTK_VERSION=-gtk3
-		AC_CHECK_LIB(webkitgtk-3.0, JSContextGroupCreate,,
-	                AC_CHECK_LIB(javascriptcoregtk-3.0, JSContextGroupCreate,WEBKIT_PC=javascriptcoregtk-3.0)) 
+		AC_CHECK_LIB(webkit2gtk-4.0, JSContextGroupCreate,,
+	                AC_CHECK_LIB(javascriptcoregtk-4.0, JSContextGroupCreate,WEBKIT_PC=javascriptcoregtk-4.0)) 
 	],
 	[1.0], [
 		AC_CHECK_LIB(webkitgtk-1.0, JSContextGroupCreate,WEBKIT_PC=webkitgtk-1.0,
